src/corelibs/U2Core/src/datatype/msa/MultipleChromatogramAlignmentRow.cpp

```diff
@@ -346,6 +352,23 @@ qint64 MultipleChromatogramAlignmentRowData::getBaseCount(qint64 before) const {
     return MsaRowUtils::getUngappedPosition(gaps, sequence.length(), trimmedRowPos, true);
 }
 
+QPair<DNAChromatogram::ChromatogramTraceAndValue, DNAChromatogram::ChromatogramTraceAndValue>
+                MultipleChromatogramAlignmentRowData::getTwoHighestPeaks(const qint64 position) const {
+    const int baseCall = chromatogram.baseCalls[position];
+    QList<DNAChromatogram::ChromatogramTraceAndValue> peaks =
+                                    { { DNAChromatogram::Trace::Trace_A, chromatogram.A[baseCall] },
+                                      { DNAChromatogram::Trace::Trace_C, chromatogram.C[baseCall] },
+                                      { DNAChromatogram::Trace::Trace_G, chromatogram.G[baseCall] },
+                                      { DNAChromatogram::Trace::Trace_T, chromatogram.T[baseCall] } };
+    std::sort(peaks.begin(),
+              peaks.end(),
+        [](const DNAChromatogram::ChromatogramTraceAndValue& first,
```

Comment:
Done

Reply:
At least C++14 must be enabled on Linux and macOS

Reply:
Lets update the compatibility level in our config if needed.

We should care about the minimal supported compiler version here, not about the C++ standard version, because different version of compilers supports different sub-sets of each standard.

For example for Linux we use gcc 5.4 (Ubuntu 16.04) as a minimal version. If this version supports the part of C++14 needed for the fix lets just change the constant in our build files (can be a separate PR).

Reply:
OK, I'll try